app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,84 @@
+class PaymentsController < ApplicationController
+  before_action :set_payment, only: %i[show edit update destroy]
+  before_action :category_params, only: %i[destroy]
+
+  # GET /payments or /payments.json
+  def index
+    @payments = Payment.all
+  end
+
+  # GET /payments/1 or /payments/1.json
+  def show; end
+
+  # GET /payments/new
+  def new
+    @payment = Payment.new
+    @user_categories = Category.where(user_id: current_user.id)
+  end
+
+  # GET /payments/1/edit
+  def edit; end
+
+  # POST /payments or /payments.json
+  def create
+    params = payment_params
+
+    @payment = Payment.new(name: params[:name], amount: params[:amount])
```

Comment:
[OPTIONAL]: I will suggest you add an image and also align the date created as in the original design.